project/AkkaDependency.scala

```diff
@@ -14,7 +14,7 @@ object AkkaDependency {
   // else if akka version is "default", then the hard coded default will be used (jenkins doesn't allow empty values for config axis)
   // else if akka.version is anything else, then the given version will be used
 
-  val defaultAkkaVersion = "2.5.12"
+  val defaultAkkaVersion = "2.5.13"
```

Comment:
Is this needed? Did we add something that is required for Akka HTTP? Otherwise we should be stay a behind on the Akka versions so that it still works with older versions if someone prefers that. This is even more important now when it is not a transitive dependency.

How was it? We added that mininum version check? Is that automatically picking up the version number from here or is it defined somewhere else also?

Reply:
Ah, updating the Akka version was still mentioned in RELEASING.md as a release step but indeed that might not make sense anymore (and would even be somewhat dangerous).

I don't see a version check (though I also remember discussing adding one). 10.1.1 still had version 2.5.11, perhaps we should even revert to that?

Reply:
I think it's more important to get this release out now than testing again with another version. Seems things have been tested with `2.5.12` so let's stick with that.

I think this should be followed up afterwards. Lookin to the version checking again, and document this aspect in RELEASING.

Reply:
and here is the PR for the version checking that was closed without merging https://github.com/akka/akka-http/pull/1880